### PR TITLE
fix bad commit

### PR DESCRIPTION
**Pull request by Xakki**
This commit broke compatibility
https://github.com/symfony/console/commit/4510f04e70344d70952566e4262a0b11df39cb10#diff-25a75e093c4a0d5564a36cebbb1384b3R461

### CODE REVIEW

File: src/Phinx/Console/Command/Init.php [hunk 1]
```diff
@@ -38,13 +38,14 @@
...
-
+    private $name = 'init';
+    
     /**
      * {@inheritdoc}
      */
     protected function configure()
     {
-        $this->setName($this->getName() ?: 'init')
+        $this->setName($this->getName())
```
Comment (dereuromark) [inline src/Phinx/Console/Command/Init.php]: why not just `$this->setName('init')`?
After all, this current getter setter thing is super weird.

Comment (Xakki) [reply src/Phinx/Console/Command/Init.php]: For compatibility

Comment (dereuromark) [reply src/Phinx/Console/Command/Init.php]: What compatibility?

**Comment by dereuromark** [comment]
I asked them to revert their BC breaking and bad change: https://github.com/symfony/console/commit/4510f04e70344d70952566e4262a0b11df39cb10#commitcomment-34845639

**Comment by Xakki** [comment]
> I asked them to revert their BC breaking and bad change: [symfony/console@4510f04#commitcomment-34845639](https://github.com/symfony/console/commit/4510f04e70344d70952566e4262a0b11df39cb10#commitcomment-34845639)
> With private usage this easily becomes BC breaking on a major scale (as you cant reach the attr).

This option of course better.

**Comment by dereuromark** [comment]
In the meantime we have a PR that solves it also future proof:
https://github.com/cakephp/phinx/pull/1588

Closing this then in favor of the other PR.

**Pull request closed without merging**